lib/appsignal/probes/erlang_probe.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Appsignal.Probes.ErlangProbe do
+  @appsignal Application.get_env(:appsignal, :appsignal, Appsignal)
+
+  def call do
+    io_metrics()
+    scheduler_metrics()
+    process_metrics()
+    memory_metrics()
+  end
+
+  defp io_metrics do
+    {{:input, input}, {:output, output}} = :erlang.statistics(:io)
+    @appsignal.set_gauge("erlang_io", Kernel.div(input, 1024), %{type: "input"})
+    @appsignal.set_gauge("erlang_io", Kernel.div(output, 1024), %{kind: "output"})
```

Comment:
@jvanbaarsen can you change the `kind` tag keys to `type` for all metrics so they're consistent?

Reply:
Hm, something must have gone wrong with rebasing :/ Will do!